### ISSUE TITLE
Чи повинно з’явитись попередження про нове підключення на сторінці аукціону, якщо зайти на сторінку аукціону з іншого браузеру?

### ISSUE THREAD

**Issue by yuliyaSoftserve** [openprocurement/frontend#95]
У тестовому сценарії: http://ppr.org.ua/files/TestScripts_v1.00_Phase2.pdf,
ДЕНЬ 4, 110 – Участь у аукціоні, ст.7,
зазначено, що якщо зайти на сторінку аукціону з іншого пристрою або з іншого браузеру, має з’явитись попередження про нове підключення.

**Comment by myroslav** [comment]
Так, повинно з'являтися, якщо аукціон відкривається не для перегляду (анонімного) а для участі.
